UnityProject/Assets/Scripts/Health/RespiratoryHealth/RespiratorySystem.cs

```diff
@@ -159,26 +162,43 @@ private float HandleBreathing(GasMix breathGasMix)
 		}
 		else
 		{
+			IsSuffocating = false;
 			oxygenUsed = breathGasMix.GetMoles(Gas.Oxygen);
 			bloodSystem.OxygenLevel += 30;
 		}
 
 		return oxygenUsed;
 	}
 
+	public enum PressureChecker {
+		noAlert = 1,
+		tooHigh = 2,
+		tooLow = 4
+	}
+
 	private void CheckPressureDamage(float pressure)
 	{
 		if (pressure < AtmosConstants.MINIMUM_OXYGEN_PRESSURE)
-		{
+		{	
+			//1 pressure status indicates Low Pressure
+			pressureStatus |= PressureChecker.tooLow;
+			pressureStatus &= ~(PressureChecker.noAlert);
 			ApplyDamage(AtmosConstants.LOW_PRESSURE_DAMAGE, DamageType.Brute);
 		}
 		else if (pressure > AtmosConstants.HAZARD_HIGH_PRESSURE)
 		{
 			float damage = Mathf.Min(((pressure / AtmosConstants.HAZARD_HIGH_PRESSURE) - 1) * AtmosConstants.PRESSURE_DAMAGE_COEFFICIENT,
 				AtmosConstants.MAX_HIGH_PRESSURE_DAMAGE);
-
+			//2 pressure status indicates High Pressure
+			pressureStatus |= PressureChecker.tooHigh;
+			pressureStatus &= ~(PressureChecker.noAlert);
 			ApplyDamage(damage, DamageType.Brute);
 		}
+		else
+		{
+			pressureStatus |= PressureChecker.noAlert;
```

Comment:
Just set the enum value directly, that's essentially what you're doing anyways and it's a lot easier to read